src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegistrationSection/ProgramSelector/ComposedProgramSelector.component.tsx

```diff
@@ -15,8 +13,10 @@ import {
     withFilterProps,
 } from '../../../../../../FormFields/New';
 import { NonBundledDhis2Icon } from '../../../../../../NonBundledDhis2Icon';
+import type { Program } from '../../../../../../../metaData';
+import type { ComposedProgramSelectorProps, ProgramOption } from './ProgramSelector.types';
 
-const getStyles = (theme: Theme) => ({
+const styles = createStyles({
```

Comment:
In the cases where you actually need the theme, please keep the same code and override the export using `as ComponentType<Props>` Like we used to do.